README.md

```diff
@@ -312,6 +313,11 @@ on:
     branches:
     - master
   pull_request:
```

Comment:
```suggestion
```
This should be removed, the API will only accept results from the default branch.

Reply:
@DuncanCasteleyn

You're correct, good catch!

Would creating a separate workflow even make sense in this case? Since, as you said, the API will only send the results to GitHub Dependency Graph if you're running it on the default branch.

Reply:
A separate workflow to verify if a PR submits new vulnerabilities I mean, so for the SBOM submitting it only makes sense to use on the default branch.

Reply:
@DuncanCasteleyn

Got it, thanks for clarifying. Indeed, that would be better.

Just to confirm, should this example workflow be added to this PR?

Reply:
I think there's are already examples present for that kind of workflow.